tests/ImageSharp.Benchmarks/Codecs/EncodeIndexedPng.cs

```diff
@@ -85,7 +85,7 @@ public void PngCoreWu()
         public void PngCoreWuNoDither()
         {
             using var memoryStream = new MemoryStream();
-            var options = new PngEncoder { Quantizer = new WuQuantizer(new QuantizerOptions { Dither = null }) };
+            var options = new PngEncoder { Quantizer = new WuQuantizer(new QuantizerOptions { Dither = null }), ColorType = PngColorType.Palette };
```

Comment:
I made this change so I can get perf coverage for it. Should I keep it or revert it?

Reply:
Yep. That's an oversight from us. 

Very happy to see what other optimizations you can bring as long as they are well commented. Thanks! 👍

Reply:
Sure, happy to contribute. I think it will be more productive for us to contribute or investigate the performance issues if these benchmarks are in our system. Any progress on https://github.com/SixLabors/ImageSharp/discussions/1795?

Reply:
Leaving that one for @antonfirsov Likely after we complete #1730 and release V2.

Reply:
Answered:
https://github.com/SixLabors/ImageSharp/discussions/1795#discussioncomment-1635798